schema.cc

```diff
@@ -1374,6 +1374,22 @@ schema::regular_columns() const {
             , _raw._columns.end());
 }
 
+schema::const_iterator_range_type
+schema::columns(column_kind kind) const {
+    switch (kind) {
+    case column_kind::partition_key:
+        return partition_key_columns();
+    case column_kind::clustering_key:
+        return clustering_key_columns();
+    case column_kind::static_column:
+        return static_columns();
+    case column_kind::regular_column:
+        return regular_columns();
+    default:
+        std::abort();
```

Comment:
Remove this please. Without the default compiler should complain when switch does not cover all the values of the enum. With default the check is postponed to runtime.

Reply:
You mean the `default:` branch, not the entire switch